sdcm/sct_config.py

```diff
@@ -1218,6 +1218,14 @@ def __init__(self):
                     f'Test duration too long for spot_duration instance type. ' \
                     f'Max possible test duration time for this instance type is {MAX_SPOT_DURATION_TIME} minutes'
 
+        # 11) validate authenticator parameters
+        if self.get('authenticator') and self.get('authenticator').lower() == "passwordauthenticator":
```

Comment:
The password authenticator class should be exactly the same as expected in the `scylla.yaml`

Reply:
fixed